roles/picksome/files/picksome.php

```diff
@@ -0,0 +1,67 @@
+<?php
+# While this next line should be taken care of in extenions, and indeed
+# it is, because this extension is loaded via wfLoadExtension, and because
+# Collection is loaded the old way, the sidebar before output is called
+# preferring Collection, placing it higher on the siderbar than PickSome.
+#
+# This was requested to be changed, and the most straightforward way is
+# to add it to the hook directly when loaded by LocalSettings
+$wgHooks['SidebarBeforeOutput'][] = 'PickSomeHooks::onSidebarBeforeOutput';
+wfLoadExtension('PickSome');
+$wgPickSomeNumberOfPicks = 15;
+
+$wgLFCPickSomeEligiblePage = "TorqueConfig:ValidProposals";
+
+# This may not be performant and require caching if number of users/page hits
```

Comment:
"This may not be performant, and it may require caching if number of users/page hits becomes large."